.gitignore

```diff
@@ -100,3 +100,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+
+deploy.bat
```

Comment:
This looks like a change that's specific to your workspace?

Reply:
Yes it is.